DDD-13.md

```diff
@@ -0,0 +1,595 @@
+        "group": "Connection",
+        "groupOrder": 0,
+        "width": "medium",
+        "importance": "high"
+      },
+      "validation": [
+        {
+          "type": "regex",
+          "pattern": "^[a-zA-Z0-9._-]+$",
+          "message": "Only alphanumeric characters, hyphens, dots and underscores are allowed"
+        }
+      ]
+    },
+    {
+      "name": "table.include.list",
+      "type": "list",
+      "display": {
+        "label": "Include Tables",
+        "description": "The tables for which changes are to be captured",
+        "group": "Filters",
+        "groupOrder": 2,
+        "width": "long",
+        "importance": "high"
+      },
+      "validation": [
+        {
+          "type": "regex-list",
+          "message": "Each item must be a valid regular expression"
+        }
+      ]
+    },
+     {
+        "name": "connection.adapter",
+        "type": "string",
+        "display": {
+           "label": "Connector adapter",
+           "description": "The adapter to use when capturing changes from the database.",
+           "group": "Connection advanced",
+           "groupOrder": 1,
+           "width": "medium",
+           "importance": "high"
+        },
+        "valueDependants": [
+           {
+              "values": ["LogMiner"],
+              "dependants": ["log.mining.buffer.type"]
+           }
+        ],
+        "validation": [
+           {
+              "type": "regex",
+              "pattern": "[XStream|LogMiner|LogMiner_Unbuffered|OLR]",
+              "message": "Permitted value are: XStream, LogMiner, LogMiner_Unbuffered, OLR"
+           }
+        ]
+     },
+     {
+        "name": "log.mining.buffer.type",
+        "type": "string",
+        "display": {
+           "label": "Controls which buffer type implementation to be used",
+           "description": "The buffer type controls how the connector manages buffering transaction data.",
+           "group": "Connection advanced",
+           "groupOrder": 2,
+           "width": "medium",
+           "importance": "low"
+        },
+        "valueDependants": [
+           {
+              "values": ["ehcache"],
+              "dependants": [
+                 "log.mining.buffer.ehcache.global.config",
+                 "log.mining.buffer.ehcache.transactions.config",
+                 "log.mining.buffer.ehcache.processedtransactions.config",
+                 "log.mining.buffer.ehcache.schemachanges.config",
+                 "log.mining.buffer.ehcache.events.config"
+              ]
+           },
+           {
+              "values": ["infinispan_embedded", "infinispan_remote"],
```

Comment:
Seems to me that we have the same kink of information just described in different ways and `dependents` seems to be another one 

We have:
* the `group` that should group similar configuration, why not us it for example for grouping all `adapter` configs?
* the prefixes, for example `log.miner.*` is a kind of groups and also creating "dependencies". Isn't it?
* dependents that create a relation between configs, isn't this doable with `prefixes` or `groups`?

@jpechane @Naros WDYT?